### PR TITLE
GCE provisioning

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: -v --long-running --use-provider ec2 cfme/tests/services/test_cloud_service_catalogs.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 89a5841bd68cbf0aeb838cf3932159c0f67f349d:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9b5d065de8fe7041923e20d251befc3d9ebca8b6:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 100fc1e1689fff623895b49941d71bb14c27d2ea:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d0423b217b11379a96bfa4bc8adfbb6732d36e34:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 23b1f6335d15ecb220d41d480bc0da92b6653c38:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cc70849fec2aa844ddfa6191ad75fb75c2056846:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2e56cb2146250a8a28f215ae5d2482fd138d56ef:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 10135cf6fa4ad2e40c55538ea9cc79418a310c53:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/services/test_cloud_service_catalogs.py [hunk 1]
```diff
@@ -57,14 +58,16 @@ def catalog():
     yield catalog
 
 
-@pytest.mark.meta(blockers=[1344103, 1367176])
+@pytest.mark.meta(blockers=[BZ(1379476, forced_streams=["5.7", "upstream"])])
```
Comment (psav) [inline cfme/tests/services/test_cloud_service_catalogs.py]: This currently skips on ALL streams, is that right?

Comment (sshveta) [inline cfme/tests/services/test_cloud_service_catalogs.py]: Yes the bug is for all streams , however it applies only to GCE provider

Comment (psav) [inline cfme/tests/services/test_cloud_service_catalogs.py]: OK, well hte way it's working right now it's skipping them for all providers

**Comment by dajoRH** [comment]
Lint report for commit 864eb75ac2df3d7852c1c465fbb22de128dfe724:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by sshveta** [comment]
The Openstack service request fails for error : "{\"badRequest\": {\"message\": \"The requested availability zone is not available\", \"code\": 400}}"
Openstack error , dajo is looking into it .

**Comment by dajoRH** [comment]
Lint report for commit 484d5bc1d70a355f00c4b18162ffd967e7dc182d:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 972592f768e85c91f468d1eaf0fea1874dad81e4:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**